views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss

```diff
@@ -0,0 +1,178 @@
+// =============
+// TAO specific
+// =============
+
+@import "inc/bootstrap";
+
+// Color mapping
+
+$disabledBg: whiten($websiteBorder, .3);
+$disabledTxt: black;
+$disabledTxtShadow: white(.8);
+
+$enabledBg: whiten($info, .1);
+$enabledTxt: white;
+$enabledTxtShadow: blacken($info, .2);
+
+$activeBg: whiten($activeInteraction, .1);
+$activeTxt: white;
+$activeTxtShadow: blacken($activeInteraction, .2);
+
+$record: $activeInteraction;
+$playback: whiten($info, .3);
+
+
+// =============
+// Interaction
+// =============
+
+.qti-customInteraction {
+  .audioRecordingInteraction {
+
+    padding: 10px;
+
+    // input meter
+
+    .input-meter {
+      border: 2px solid whiten($disabledBg, .5);
+      border-radius: 10px;
+      width: 58px;
+      padding: 10px;
+      margin-right: 10px;
+      margin-top: 2px;
+      float: left;
+
+      display: flex;
+
+      &.record {
+        border-color: $activeBg;
+        background-color: whiten($activeBg, 0.9);
+      }
+
+      .leds {
+        flex: 0 0 10px;
+      }
+
+      .mic {
+        flex: 0 0 28px;
+
+        img {
+          height: 28px;
+          width: 28px;
+          margin-top: 10px;
+        }
+      }
+    }
+
+
+    // Controls
+
+    .audiorec-control {
+      margin: 10px 10px 10px 0;
+      border-radius: 3px;
+      font-size: 14px !important;
```

Comment:
I'd rather leave it as an absolute value to avoid depending on the context, which we don't know in case of a PCI